src/polyfills.ts

```diff
@@ -10,7 +11,8 @@ const originalConsole = window.console
 // define a new console
 const proxiedConsole = new Proxy(window.console, {
   get(obj, prop: keyof Console) {
-    if (process.env.NODE_ENV !== 'production') {
+    // show logs in all environments EXCEPT production & ens
+    if (!isProd || !isEns) {
```

Comment:
well, maybe STAGING we don't want logs either

as close as PROD principle

Reply:
done in https://github.com/gnosis/cowswap/pull/2567/commits/48961abbd08b946e40341d3534d0911bd0c4ad8f